src/seedu/addressbook/ui/TextUi.java

```diff
@@ -75,7 +68,7 @@ private boolean isCommentLine(String rawInputLine) {
      * @return command (full line) entered by the user
```

Comment:
Perhaps more formatting-related stuff (e.g. COMMENT_LINE_FORMAT_REGEX) can be moved to the Formatter? May be even things like this should be moved to the Formatter? Note that it may not always be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.